tour_ui/config/tour.tour.backdrop.json

```diff
@@ -0,0 +1,206 @@
+{
+    "_config_name": "tour.tour.backdrop",
+    "title": "Backdrop",
+    "name": "backdrop",
+    "editable": 1,
+    "paths": [
+        "<front>"
+    ],
+    "description": "This is a beginners tour of Backdrop CMS. ",
+    "auto_start": 1,
+    "enabled": 1,
+    "steps": {
+        "Welcome": {
+            "title": "Welcome",
+            "text": "Welcome to this tour of your new Backdrop CMS site. We will start here on the front page and show you some of the most important things you need to know about your site. ",
+            "selector": ".block-hero",
+            "jump": "",
+            "id": "Welcome",
```

Comment:
Id's are meant to be machine names, sort of, so lower case please.

Reply:
Sure. I did manually edit the json file for many of these, but you should know that the ID's generated by the tour builder module capitalize the ID. I opened an issue. I'll fix this later in my tour.